db/migrate/20230510123420_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+    # add_index :posts, :user_id
```

Comment:
- [ ] It would be nice if you remove the commented piece of code to make your codebase clean and professional 👍 as required in the Ruby best practices 😊